### PR TITLE
#[161277640] admin/attendants should be able create an account

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What does this PR do?
Have create and get endpoints  in Users for the store-API
#### Description of Task to be completed?
Have the following endpoint working:
> POST/api/v1/signup
#### payload
```
{
       "username":"Kristine Camil",
       "password":"password",
       "email":"camil@new.com",
       "role":"attendant"
}
```

#### How should this be manually tested?
After cloning the repo locally run python run.py
. Using postman test the above endpoints using the below header
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/161277640

### CODE REVIEW

File: v1/app/views/users_views.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from flask import Blueprint
+from flask_restful import reqparse
+
+# local imports
+from app.controllers.users_controllers import Users
+
+don_user = Blueprint('users', __name__, url_prefix="/api/v1")
+
+user_insatnce = Users()
```
Comment (danuluma) [inline v1/app/views/users_views.py]: Isn't this a typo?

Comment (Ruiru11) [reply v1/app/views/users_views.py]: what part exactly?
thanks @danuluma

Comment (ghost) [reply v1/app/views/users_views.py]: @Ruiru11 Do you mean `user_instance` or `user_insatnce`? `cc:` @danuluma

Comment (Ruiru11) [reply v1/app/views/users_views.py]: it should be instance thanks

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**